spec/helpers/script_helper_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'rails_helper'
 
+def content_security_policy_nonce; end
+
 RSpec.describe ScriptHelper do
   include ScriptHelper
 
```

Comment:
This leaks the method into the global namespace. This is an option to define it on the anonymous class that Rspec makes so it still works

```suggestion
RSpec.describe ScriptHelper do
  include ScriptHelper

  def content_security_policy_nonce; end
```

Another option to make it more localized
```suggestion
RSpec.describe ScriptHelper do
  subject(:helper) do
    Class.new do
      include ScriptHelper

      def content_security_policy_nonce; end
    end.new
  end
```

and then all the tests might be needed to call `helper.javascript_include_tag_without_preload` etc or something

Reply:
Good call! I'll make those updates in the morning.

Reply:
Updated in 57be65edc with the first option. I assume "anonymous class" is specific to the file, right, not shared with other specs?

Reply:
Yup, sorry it's not actually anonymous it's named for the example names. But yes it's specific to the file